de.ient.YUView.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: de.ient.YUView
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: YUView
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --env=QT_NO_FT_CACHE=1
+
+modules:
+  - name: YUView
+    buildsystem: qmake
+    sources:
+      - type: git
+        branch: deployment
+        url: https://github.com/IENT/YUView.git
+        commit: 64ccb50b85eed1db9d44b7f26c960b0807927f33
+    build-commands:
+      - install -Dm644 images/IENT-YUView-32.png /app/share/icons/hicolor/32x32/apps/de.ient.YUView.png
+      - install -Dm644 images/IENT-YUView-64.png /app/share/icons/hicolor/64x64/apps/de.ient.YUView.png
+      - install -Dm644 images/IENT-YUView-128.png /app/share/icons/hicolor/128x128/apps/de.ient.YUView.png
+      - install -Dm644 images/IENT-YUView-256.png /app/share/icons/hicolor/256x256/apps/de.ient.YUView.png
+      - install -Dm644 images/IENT-YUView-512.png /app/share/icons/hicolor/512x512/apps/de.ient.YUView.png
+      - install -Dm644 images/IENT-YUView-1024.png /app/share/icons/hicolor/1024x1024/apps/de.ient.YUView.png
+
+  - name: libde265
```

Comment:
If YUView depends on this library, then have to move it up in the manifest.

Reply:
that library is loaded at runtime, so it's not required for building. I've been testing all these changes locally with flatpak-builder and it works. I could still move it up, but I don't think it will make a difference.